### PR TITLE
Added search icon and resolved conflict

### LINKED ISSUE #111: Adding a Search Icon in the Search bar
**Is your feature request related to a problem? Please describe.**
No, my feature request is not related to any problem. It's about making the page more attractive.

**Describe the solution you'd like**
I want to add a "Search Icon" in the "Search Bar" to give a better look to the project.

**Are you willing to contribute to this issue?** [Yes/No]
Yes

**Describe alternatives you've considered**
As of now, I have no alternatives for this, I just want to add a "Search Icon" in the "Search Bar".

**Additional context**
Screenshot (2706)_LI

@maneprajakta I want to contribute this as a part of GSSOC'21.Please assign this issue to me.

### PR DESCRIPTION
**Pull request by Sweeti-Gupta**
Fixed #111

## Description
- Search icon has been added and a fresh PR is generated to resolve conflict.
- #111 

## Affected Dependencies
N/A

## How has this been tested?
python manage.py runserver

## GIF Before
Kitabe_before

## GIF After
Kitabe_after2

## Checklist
- [x] I have followed the [Contribution Guidelines](https://github.com/Praful932/Kitabe/blob/master/CONTRIBUTING.md).
- [x] My changes do not edit/add any unrequired files.
- [x] My changes are covered by tests.

### CODE REVIEW

**Review by MrDevLovePer — APPROVED** [review]

File: templates/mainapp/layout.html [hunk 1]
```diff
@@ -141,15 +134,14 @@ <h4 class="modal-title" id="booktitle">Basic Modal</h4>
...
-	
-	<a href='#top'>
-	<button  id="topBtn" title="Go to top"><i class="fa fa-angle-up"></i></button></a>
+
+    <a href='#top'>
+        <button id="topBtn" title="Go to top"><i class="fa fa-angle-up"></i></button></a>
     <!-- End boostrap modal -->
-    {% block main %}
-    {% endblock main %}
+    {% block main %} {% endblock main %}
```
Comment (maneprajakta) [inline templates/mainapp/layout.html:141]: Unnecessary changes coming from your side

**Review by maneprajakta — CHANGES_REQUESTED** [review]
- There are Conflicts resolve them.
- Do only those changes that solve the issue.
